test/WebJobs.Script.Tests/EndToEndTestsBase.cs

```diff
@@ -44,5 +48,87 @@ public EndToEndTestsBase(TTestFixture fixture)
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace("script processed queue message")));
             Assert.True(trace.Contains(TestHelpers.RemoveByteOrderMarkAndWhitespace(messageContent)));
         }
+
+        protected async Task EasyTablesTest(bool writeToQueue = true)
+        {
+            // EasyTables needs the following environment vars:
+            // "AzureWebJobsMobileAppUri" - the URI to the mobile app
```

Comment:
So these tests now require this. Perhaps we can create a single MobileService in our test subscription that we can all share? Are all your tests isolated enough that we could run concurrently against the same service?